src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const avarageTime = 3;
```

Comment:
Typo in variable name `avarageTime`. It should be `averageTime`. Variable names should be spelled correctly to avoid confusion.